Source/IGListStackedSectionController.m

```diff
@@ -225,6 +225,14 @@ - (void)deselectItemAtIndex:(NSInteger)index sectionController:(IGListSectionCon
     [self.collectionContext deselectItemAtIndex:offsetIndex sectionController:self animated:animated];
 }
 
+- (void)selectItemAtIndex:(NSInteger)index
+        sectionController:(IGListSectionController *)sectionController
+                 animated:(BOOL)animated
+           scrollPosition:(UICollectionViewScrollPosition)scrollPosition {
+    const NSInteger offsetIndex = [self relativeIndexForSectionController:sectionController fromLocalIndex:index];
+    [self.collectionContext selectItemAtIndex:offsetIndex sectionController:self animated:animated scrollPosition:scrollPosition];
```

Comment:
@MarvinNazari can we get a unit test for this too? Once we have that we're good to go!